ui/packages/catalog-ui-search/src/main/webapp/component/relative-time/relative-time.view.js

```diff
@@ -75,21 +75,17 @@ module.exports = Marionette.LayoutView.extend({
     return `RELATIVE(${duration})`
   },
   parseValue(value) {
-    if (
-      value === null ||
-      value === undefined ||
-      value.indexOf('RELATIVE') !== 0
-    ) {
+    if (!value)
+      return
+
+    const match = value.match(/RELATIVE\(Z?(.*)(\d+\.*\d*)(.)\)/)
```

Comment:
❓ Why is `.*` in its own capture group here? Wouldn't `\(Z?.*(\d+\.*\d*)(.)\)/)`have the same effect since you're using `*`?

Reply:
I don't think the regex you suggested is equivalent

Reply:
I ended up changing the` .` to `[A-Z]`